packages/viewer-sandbox/src/type-augmentations/three.d.ts

```diff
@@ -0,0 +1,8 @@
+import Three from 'three'
+
+declare module 'three' {
+  interface Object3D {
+    // TODO: @alex - does this really exist? it's not on the three.js type
+    material: unknown
```

Comment:
in Sandbox.ts the `object.material as SpeckleLineMaterial` line relies on a nonexistant Object3D property, I augmented the type, but @AlexandruPopovici you should ensure that it really does exist

Reply:
Oh, you were referring to the `material` property. I thought you meant `Object3D`